pkg/backend/httpstate/mock.go

```diff
@@ -41,6 +41,7 @@ type MockHTTPBackend struct {
 		req apitype.CreateDeploymentRequest,
 		opts display.Options,
 		deploymentInitiator string,
+		streamDeploymentLogs bool,
```

Comment:
We should probably rename this as well to `suppressStreamLogs`.  Sorry I forgot pointing this out earlier.

Reply:
ah damn! good catch